src/CDEngineSDKTemplates/CDEngineSDKTemplates.project.cdeHostProject/cdeHostProject.vstemplate

```diff
@@ -1,21 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <VSTemplate Version="3.0.0" Type="Project" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
-    <Name>Console Host App for C-Labs C-DEngine (.NET Framework)</Name>
+    <Name>Console Application Host for C-Labs C-DEngine (.NET Framework)</Name>
     <Description>A project template that creates a Console Application to host the C-Labs C-DEngine IoT communication relay engine.</Description>
```

Comment:
Can we remove "C-Labs" here? This is in other vstemplates as well and since its open source now it makes more sense to remove the company name

Reply:
Yeah, good point. I'll clean this up.